addRider.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+use Collection\HTML\HeadHtml;
+use Collection\HTML\PageContents\AddRiderHtml;
+use Collection\Models\RidersModel;
+
+require_once 'database.php';
+require_once 'vendor/autoload.php';
+
+$ridersModel = new RidersModel($db);
+$headHtml = new HeadHtml();
+$addRiderHtml = new AddRiderHtml();
+
+// Handling form submission
+$name = $_POST['name'] ?? false;
+$image = $_POST['image'] ?? false;
+$team = $_POST['team'] ?? false;
+$nationality = $_POST['nationality'] ?? false;
+$dob = $_POST['dob'] ?? false;
+$giroGcWins = $_POST['giroGcWins'] ?? false;
+$tourGcWins = $_POST['tourGcWins'] ?? false;
+$vueltaGcWins = $_POST['vueltaGcWins'] ?? false;
+$giroStageWins = $_POST['giroStageWins'] ?? false;
+$tourStageWins = $_POST['tourStageWins'] ?? false;
+$vueltaStageWins = $_POST['vueltaStageWins'] ?? false;
+
+if (strlen($name) > 0) {
+    $nameValidation = true;
+} else {
+    $nameValidation = false;
+}
+
+if (strlen($image) > 0) {
+    $imageValidation = true;
+} else {
+    $imageValidation = false;
+}
+
+if (strlen($team) > 0) {
+    $teamValidation = true;
+} else {
+    $teamValidation = false;
+}
+
+if (strlen($nationality) > 0) {
+    $nationalityValidation = true;
+} else {
+    $nationalityValidation = false;
+}
+
+if (strlen($dob) === 10) {
+    $dobValidation = true;
+} else {
+    $dobValidation = false;
+}
```

Comment:
Nice validation! This is the type of thing that should go into it's own function/object. It's likely you would want the exact same validation rules when editing a rider. It's also something you could then unit test.

Reply:
Created RiderFormValidator class + unit tested it- are the tests ok though?